test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.cs

```diff
@@ -3,71 +3,56 @@
 
 namespace Microsoft.VisualStudio.TestPlatform.TestFramework.UnitTests;
 
-extern alias FrameworkV1;
-extern alias FrameworkV2;
-
 using System;
 using System.Globalization;
 using System.Threading.Tasks;
-using MSTestAdapter.TestUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-using Assert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
-using StringAssert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.StringAssert;
-using TestClass = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute;
-using TestFrameworkV2 = FrameworkV2.Microsoft.VisualStudio.TestTools.UnitTesting;
-using TestMethod = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute;
+using global::TestFramework.ForTestingMSTest;
 
-[TestClass]
 public partial class AssertTests
 {
     #region That tests
-
-    [TestMethod]
     public void ThatShouldReturnAnInstanceOfAssert()
     {
-        Assert.IsNotNull(TestFrameworkV2.Assert.That);
+        Verify(Assert.That is not null);
     }
 
-    [TestMethod]
     public void ThatShouldCacheAssertInstance()
     {
-        Assert.AreEqual(TestFrameworkV2.Assert.That, TestFrameworkV2.Assert.That);
-    }
+        Verify(Assert.That == Assert.That);
```

Comment:
It's equivalent to what was there before but maybe we could refactor it to do:

```suggestion
        Verify(object.ReferenceEquals(Assert.That, Assert.That));
```